examples/go/echo/echo.go

```diff
@@ -5,13 +5,21 @@ import (
 	"context"
 	"fmt"
 
+	"ftl/builtin"
 	"ftl/time"
 
 	"github.com/TBD54566975/ftl/go-runtime/ftl"
 )
 
 var defaultName = ftl.Config[string]("default")
 
+type EchoEvent struct {
+	Message string
+}
+
+var Echotopic = ftl.Topic[EchoEvent]("echotopic")
+var sub = ftl.Subscription(Echotopic, "sub")
```

Comment:
Do we want these added to `echo` at this point? `echo` might get quite complicated if we add all the different types here. We likely want some new "kitchen sink" examples at some point in the future.

Reply:
I was planning on reverting this file's changes before merging, but figured I should leave it in for review so anyone reading can see how these values go mapped to the actual otel stream output. Having an exemplar package would be lovely... I was definitely missing that today hehe

Reply:
That's probably something we can do this week since it won't impact the stabilization work (and might help us find more bugs)

Reply:
pasting it here so we can find it later: https://github.com/TBD54566975/ftl/issues/1625